hooks/024-configure-bootchart.chroot

```diff
@@ -0,0 +1,71 @@
+#! /bin/sh -ex
```

Comment:
None of this should be a hook, but simply static files in static/ no?

I guess we can do that separately.